slurm-cookbook/docker/hello.job

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
The batch file is created as one of the tutorial steps. Do you need it here?

Reply:
Hm. I guess not.